src/matlab2tikz.m

```diff
@@ -3142,8 +3142,13 @@ function assertRegularAxes(handle)
     [m2t, pos1, pos2] = getPositionOfArrow(m2t, handle);
 
     styleOpts = opts_print(drawOptions);
-    str       = sprintf('\\draw[-{Stealth}, %s] %s -- %s;\n', ...
-                        styleOpts, pos1, pos2);
+    if isa(handle,'matlab.graphics.shape.DoubleEndArrow')
```

Comment:
While correct, I think it would be better to do something like:
```matlab
if isa(handle, 'matlab.graphics.shape.DoubleEndArrow')
    arrowSpec = '{Stealth}-{Stealth}';
else
    arrowSpec = '-{Stealth}';
end
```
and then either insert that `arrowSpec` into `drawOptions` (as the very first thing after `opts_new`), or add it via a `'%s'`  option into the output string.

That would reduce a bit of duplicate code and possibilities for formatting problems in the future.